src/tapis-ui/src/components/streams/instruments/InstrumentList/InstrumentList.scss

```diff
@@ -0,0 +1,31 @@
+$core-portal-color-primary: #9d85ef;
+
+.instrument-list {
```

Comment:
Use css modules rather than a global css, so that there can be role-based namespacing for css styles that only appear within one element.